google-cloud-logging/lib/google/cloud/logging/v2/logging_service_v2_client.rb

```diff
@@ -140,10 +154,6 @@ def self.match_log_from_log_name log_name
           #   or the specified config is missing data points.
           # @param timeout [Numeric]
           #   The default timeout, in seconds, for calls made through this client.
-          # @param app_name [String]
```

Comment:
These fields are for internal-use only (i.e. only for your manual layers to use). We don't want to encourage general usage, because they won't do anything useful for arbitrary values.